logic/mdwLogic.js

```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
+const logger = require('../utils/logger')(module);
+
+const LIMIT = 100; // max 1000
+const knownHashes = {};
+
+module.exports = class MdwLogic {
+  // fetches pages forwards, if no next its the last page, don't cache that
+  static async iterateMdw(next, abortIfHashKnown = false) {
```

Comment:
I would argue we should default `abortIfHashKnown` to `true`

Reply:
no, as this is only used for backwards fetch of events, iterateMdw is also used for names